mods/ra/weapons/missiles.yaml

```diff
@@ -123,6 +125,8 @@ MammothTusk:
 		HorizontalRateOfTurn: 60
 		RangeLimit: 9c614
 	Warhead@1Dam: SpreadDamage
+		ValidTargets: AirborneActor, Infantry
+		InvalidTargets: Vehicle
```

Comment:
Hidden balance change?

Reply:
Ah no. I just needed to add something to overwrite InvalidTargets: property, for all it matters it might as well be structures

Reply:
I think that might change the behavior as well.

Reply:
I could list all the non-infantry ground target types if it makes it better

Reply:
Maybe `NoAutoTarget`

Reply:
I refactored to not use InvalidTargets: at all. It's completely unnecessary

Reply:
Nevermind, actually. I thought mammoth would use rockets when attacking buildings, but I checked bleed and it doesn't.